.github/workflows/jekyll-build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Jekyll site
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@0a29871fe2b0200a17a4497bae54fe5df0d973aa # v1.115.3
+        with:
+          ruby-version: '3.0' # Not needed with a .ruby-version file
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+          cache-version: 0 # Increment this number if you need to re-download cached gems
+      - name: Build with Jekyll
+        # Outputs to the './_site' directory by default
+        run: bundle exec jekyll build --baseurl "${{ steps.pages.outputs.base_path }}"
```

Comment:
steps.pages doesn't exist.
Move
```yml
      - name: Setup Pages
        id: pages
        uses: actions/configure-pages@v2
```
from `jekyll-deploy.yml` here, that step is used to configure jekyll/other before the build?